samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -18,14 +18,19 @@
 #define DEVICE_NAME     CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
 #define NAME_LEN        30
-#define TIMEOUT_SYNC_CREATE K_SECONDS(10)
+#define PEER_NAME_LEN_MAX 30
+/* BT Core 5.3 specification allows controller to wait 6 periodic advertising events for
+ * synchronization establishment, hence timeout must be longer than that.
+ */
```

Comment:
Can you specific the section where this is stated as well?

Reply:
section added